theia-turn/turn-sprotty/src/views/ucm/endpoint.node.tsx

```diff
@@ -10,8 +10,9 @@ export class EndPointNodeView implements IView {
     static readonly TYPE = 'turnnode:endpoint';
 
     render(node: TURNNode, context: RenderingContext): VNode {
+        const y = Math.max(node.bounds.height, 0) - node.getDimension();
```

Comment:
if 0 < `node.bounds.height` < `node.getDimension()` y would be negative.
Suggestion: `const yOffset = Math.max(node.bounds.height - node.getDimensions(), 0);`